packages/cfd/src/Containers/cfd-password-modal.tsx

```diff
@@ -151,12 +151,7 @@ const ReviewMessageForMT5 = ({
     manual_status,
 }: TReviewMsgForMT5) => {
     if (is_selected_mt5_verified) {
-        return (
-            <Localize
-                i18n_default_text='To start trading, <0/>transfer funds from your Deriv account into this account.'
-                components={[<br key={0} />]}
-            />
-        );
+        return <Localize i18n_default_text='Enable trading with your first transfer.' />;
```

Comment:
This is applicable only if POI is POa is verified.. should we update irrespective of authentication?

Reply:
As far i know, first line of the message is static and 2nd line is based on condition.

Reply:
There was nothing mentioned about authentication in the card

Reply:
@hasan-deriv maybe we need to confirm with squad.

Reply:
sure..